src/librustc_ast_passes/feature_gate.rs

```diff
@@ -517,12 +520,18 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
 
     fn visit_generic_param(&mut self, param: &'a GenericParam) {
         match param.kind {
-            GenericParamKind::Const { .. } => gate_feature_post!(
-                &self,
-                const_generics,
-                param.ident.span,
-                "const generics are unstable"
-            ),
+            GenericParamKind::Const { .. } => {
+                // FIXME: we currently stop compilation prematurely in case a
```

Comment:
```suggestion
                // FIXME(const_generics): we currently stop compilation prematurely in case a
```
This just makes it easier to track the FIXME later.